_components/radio-buttons/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/radio-buttons/accessibility-tests/
+layout: accessibility-test
+component:
+ name: radio buttons
+title: Radio buttons accessibility tests
+category: Components
+lead: Any USWDS radio buttons component should pass these manual accessibility tests.
```

Comment:
**suggestion (non-blocking):** Recommend we simplify the text here a bit for clarity. 

```suggestion
lead: Any USWDS radio button should pass these manual accessibility tests.
```

Reply:
Resolved in [610d52e](https://github.com/uswds/uswds-site/pull/2821/commits/610d52ee302096392b62b51d30b4551a0a179ec6)